Development/src/components/MakeConnection.js

```diff
@@ -2,6 +2,7 @@ import setJSON from 'json-ptr';
 import cloneDeep from 'lodash/cloneDeep';
 import get from 'lodash/get';
 import set from 'lodash/set';
+import dataProvider from '../dataProvider';
```

Comment:
As MakeConnection is not a React functional component, hooks cannot be used. A dataProvider prop could be passed from a higher level component however?